bin/get_relative_abundance.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+import argparse
+import sys
+import re
+import pandas as pd
+
+from lib.get_headers import read_header_json
+
+def get_relative_abundance(kraken_report, species):
+    """Get the relative abundance for a species from the kraken report"""
+
+    rel_abnd = None
+    file = open(kraken_report, "r")
+    for line in file:
+        if re.search(f'{species}$', line):
+            rel_abnd = float(line.split('\t')[0].lstrip().split(' ')[0])
+            file.close()
```

Comment:
This line may not be reached, no? I would use `with open(kraken_report, "r") as file: ...` to be safer and not rely on Python closing the file for us (I also imagine someone might copy this code in the future where an open file may cause problems).

Reply:
That's funny because I copied this code from Tim, so I did exactly this! Good idea.

Reply:
Sorry, I meant putting `for line in file: ...` inside `with open(kraken_report, "r") as file: ...` so that the file is closed automatically. Ie

```
with open(kraken_report, "r") as file:
   for line in file:
      if re.search(f'{species}$', line):
         rel_abnd = float(line.split('\t')[0].lstrip().split(' ')[0])
         break